crates/sui-core/src/authority.rs

```diff
@@ -1071,6 +1071,8 @@ impl AuthorityState {
             adapter::new_move_vm(
                 epoch_store.native_functions().clone(),
                 epoch_store.protocol_config(),
+                self.expensive_safety_check_config
```

Comment:
Are these checks expected to activate in dry runs?

Reply:
Fair point--these should only ever activate for malicious code, no point in paying the cost for dry run.